webapp/src/js/services/live-list.js

```diff
@@ -51,8 +52,15 @@ angular.module('inboxServices').factory('LiveListConfig',
           if (c1.simprints && c2.simprints) {
             return c2.simprints.confidence - c1.simprints.confidence;
           }
-          if (c1.type !== c2.type) {
-            return ContactSchema.getTypes().indexOf(c1.type) - ContactSchema.getTypes().indexOf(c2.type);
+          const c1Type = c1.contact_type || c1.type;
+          const c2Type = c2.contact_type || c2.type;
+          if (c1Type !== c2Type) {
+            const c1TypeIndex = HARDCODED_CONTACT_TYPES.indexOf(c1Type);
+            const c2TypeIndex = HARDCODED_CONTACT_TYPES.indexOf(c2Type);
+            if (c1TypeIndex !== c2TypeIndex) {
+              return c2TypeIndex - c1TypeIndex;
```

Comment:
These should be reversed: `c1TypeIndex - c2TypeIndex`, otherwise you're sorting descending.

Reply:
Right you are!